lib/components/map/default-map.tsx

```diff
@@ -269,31 +279,44 @@ class DefaultMap extends Component {
     return (
       <MapContainer>
         <BaseMap
-          baseLayers={baseLayersWithNames}
+          // Only 1 base layer is supported at the moment
+          baseLayer={baseLayersWithNames?.[0]?.url}
           center={center}
+          mapLibreProps={{ reuseMaps: true }}
           maxZoom={maxZoom}
-          onClick={this.onMapClick}
+          onContextMenu={this.onMapClick}
```

Comment:
Should the original "on click" behavior be used instead?

Reply:
Ok, please insert a comment saying using onclick would require rewriting the click handler.

Reply:
Comment added in d3575aa5c7d8ea8b45d8959c9d74f64ac03b0274